modules/math/shared/src/main/scala/gsp/math/Wavelength.scala

```diff
@@ -79,25 +80,19 @@ object Wavelength {
   }
 
   /**
-   * Creates a Wavelength from an Int in Å.
-   * @param Å in the range (0 to 21474836)
-   * @return a Wavelength provided Å is in range
+   * Returns a `UnitConverter` for angstroms in the range 0 to 21474836.
    */
   def fromAngstroms: UnitConverter =
```

Comment:
While we're here is there any reason for these to not be `val`s?

Reply:
Yeah that makes sense.